docs/site/pages/assets/reference.mdx

```diff
@@ -22,7 +22,7 @@ const reactPlayer = new ReactPlayer({
 
 ### Storybook
 
-The reference assets + React player can be viewed in [Storybook](/storybook-demo)
+The reference assets + React player can be viewed in [Storybook](/latest/storybook-demo)
```

Comment:
Since docs can be deployed to `/latest/...`, `/next/...` or `/<major version>/...` the url would need to account for that

Reply:
@KetanReddy any suggestions on how to handle this dynamic URL then ?

Reply:
@KetanReddy @adierkens can you please give some guidance on the above ?